pages/admin/trusts/add-a-trust.js

```diff
@@ -40,7 +40,7 @@ const AddATrust = () => {
     }
 
     if (onSubmitErrors.length === 0) {
-      const submitTrust = async (name) => {
+      const submitTrust = async () => {
```

Comment:
Removing the `name` argument from this function will cause the response body to be `undefined` - is this the right behaviour? Line 50 uses the parameter

Reply:
Yeah, 'name' is defined as a state variable as at higher scope level so passing it through as a parameter to the function is unnecessary (also caused a lint warning).